capa/features/extractors/vmray/__init__.py

```diff
@@ -128,41 +147,42 @@ def _compute_sections(self):
             for elffile_section in self.sample_file_static_data.elf.sections:
                 self.sections[elffile_section.header.sh_addr] = elffile_section.header.sh_name
 
-    def _compute_process_ids(self):
+    def _compute_monitor_processes(self):
         for process in self.sv2.processes.values():
```

Comment:
do we even need to process the summary json here now?

Reply:
We currently don't emit features at the process level, only function calls which are recorded in `flog.xml`. So, as of now, the answer is no we probably don't, but to future proof the code we'll need to process both `SummaryV2.json` and `flog.xml` to achieve full coverage at the process scope, so I suggest we keep it. Please reopen if you feel strongly otherwise.